build_project/post_build.go

```diff
@@ -0,0 +1,42 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func PostRun(projectCtx *project.ProjectCtx) error {
```

Comment:
PostBuildRun?

Reply:
It's called as `build.PostRun`. Now there are `build.Run` and `build.PostRun`. Isn't it OK?

Reply:
May be ok...